editor/editor_properties_array_dict.cpp

```diff
@@ -384,12 +405,7 @@ void EditorPropertyArray::_remove_pressed(int p_index) {
 	Variant array = object->get_array();
 	array.call("remove", p_index);
 
-	if (array.get_type() == Variant::ARRAY) {
-		array = array.call("duplicate");
```

Comment:
There are duplicate calls all over and I don't understand why. Some of them have a comment saying it's so undo/redo work better, but I tried commenting it out here and undo/redo still works fine for removing items.